internal/turn/client.go

```
@@ -0,0 +1,320 @@
+	}
+
+	msg := stun.New()
+	msg.SetType(CreatePermissionRequest)
+	if err := msg.NewTransactionID(); err != nil {
+		return nil, err
+	}
+
+	xorAttr := XORPeerAddressIPv6Explicit(ipv4, peerPort, msg.TransactionID)
+	msg.Add(xorAttr.Type, xorAttr.Value)
+
+	stun.NewUsername(c.username).AddTo(msg)
```

golangci-lint (2.12.2)

[error] Error return value of `(github.com/pion/stun.Username).AddTo` is not checked

(errcheck)